addons/addon-base-raas-ui/packages/base-raas-ui/src/models/aws-accounts/AwsAccount.js

```diff
@@ -66,6 +118,13 @@ const AwsAccount = types
   // eslint-disable-next-line no-unused-vars
   .views(self => ({
     // add view methods here
+    get permissionStatusDetail() {
+      console.log('IAMHERE');
```

Comment:
Nit: Remove it or change to .debug()

Reply:
ah crap...this isn't supposed to be in here at all. Good catch, thank you--I thought I had removed all of these but I guess one slipped through the cracks